src/OrchardCore.Modules/OrchardCore.Users/Controllers/AdminController.cs

```diff
@@ -506,5 +506,42 @@ public async Task<IActionResult> EditPassword(ResetPasswordViewModel model)
 
             return View(model);
         }
+
+        [HttpPost]
+        public async Task<IActionResult> Unlock(string id)
```

Comment:
And maybe add `DisableLockOut` and `EnableLockOut` to change the feature for each user.

Reply:
The `Unlock` button in the Admin is only displayed when the user is locked (the number of failed access has been reached).

Reply:
Yes, and this is why calling this should "unlock" the user, by setting the LockedEndUtc date to null.